tests/functional/raw-node/test/unsupportedChecksums.js

```diff
@@ -1,70 +0,0 @@
-const assert = require('assert');
```

Comment:
Unless an upcoming PR is will be landing support for `STREAMING-AWS4-HMAC-SHA256-PAYLOAD-TRAILER` I think we should keep this case